components/dashboard/src/data/prebuilds/prebuild-logs-emitter.ts

```diff
@@ -23,20 +29,47 @@ export function usePrebuildLogsEmitter(prebuildId: string, taskId: string) {
     useEffect(() => {
         const controller = new AbortController();
         const watch = async () => {
+            if (taskId === null) {
```

Comment:
Isn't this already handled in the branch below?

Reply:
Yes, it used to have a different loading state handling in a different revision. Will update. For context, these checks will go away after https://github.com/gitpod-io/gitpod/pull/19916.